### ISSUE TITLE
Validation accuracy of no fine-tunned VGG-16 5X?

### ISSUE THREAD

**Issue by slothkong** [yihui-he/channel-pruning#60]
I have pruned VGG-16 using only channel-pruning. However accuracy on Imagenet val data is very low:

> accuracy@1 = 0.0245398
   accuracy@5 = 0.0847419

However your released VGG-16 5X did not suffer much accuracy loss. But what was the accuracy before you performed fine-tuning? If you say is more than 60% then I will have to check my code again ㅜㅜ

**Comment by yihui-he** [comment]
read the paper
